### PR TITLE
🐢 Turtle painting

### PR DESCRIPTION
**Pull request by fonsp**
Use simple Julia code to make a painting!

### CODE REVIEW

File: src/basic/turtles.jl [hunk 1]
```diff
@@ -0,0 +1,1254 @@
...
+md"""
+## Adding color
+
+You can use `color!` to set the color of the turtle. This will determine the color of all future lines that you draw!
+"""
+
+# ╔═╡ 14c803ff-268b-48bc-90c9-faa88010f5fe
+md"""
+TODO: which colors can you use?
+"""
```
Comment (lukavdplas) [inline src/basic/turtles.jl:143]: Rainbow!

```jl
turtle_drawing() do t
	color!(t, "red")
	forward!(t, 10)
	
	penup!(t)
	left!(t, 90)
	forward!(t, 0.4)
	left!(t, 90)
	pendown!(t)

	color!(t, "orange")
	forward!(t, 10)

	penup!(t)
	left!(t, 270)
	forward!(t, 0.4)
	left!(t, 270)
	pendown!(t)

	color!(t, "yellow")
	forward!(t, 10)

	penup!(t)
	left!(t, 90)
	forward!(t, 0.4)
	left!(t, 90)
	pendown!(t)

	color!(t, "green")
	forward!(t, 10)

	penup!(t)
	left!(t, 270)
	forward!(t, 0.4)
	left!(t, 270)
	pendown!(t)

	color!(t, "turquoise")
	forward!(t, 10)

	penup!(t)
	left!(t, 90)
	forward!(t, 0.4)
	left!(t, 90)
	pendown!(t)

	color!(t, "blue")
	forward!(t, 10)

	penup!(t)
	left!(t, 270)
	forward!(t, 0.4)
	left!(t, 270)
	pendown!(t)

	color!(t, "purple")
	forward!(t, 10)
end
```

**Comment by lukavdplas** [comment]
The functions include `left!` but not `right!`, but using both is probably more intuitive.

**Comment by fonsp** [comment]
Fixed a scaling issue so now the little turtle emoji scales along with the image!

https://github.com/JuliaPluto/featured/assets/6933510/349049ea-7c5e-46cb-9664-31b95d4011ca

File: src/basic/turtles.jl [hunk 2]
```diff
@@ -0,0 +1,1254 @@
+### A Pluto.jl notebook ###
+# v0.19.37
+
+#> [frontmatter]
+#> image = "https://github.com/JuliaRegistries/General/assets/6933510/9a925232-6a75-47e7-9ab9-f384bc389602"
+#> title = "Painting with Turtles"
+#> date = "2024-01-23"
+#> tags = ["krat", "turtle", "basic"]
```
[the remark this answers is not in the document]
Comment (fonsp) [reply src/basic/turtles.jl:8]: Let's leave it in `basic` and once we have a second notebook that goes together nicely in a category we can change it :)

Maybe a "Learning Julia" category together with "Basic mathematics with pizzas"?

**Comment by behinger** [comment]
oh my god, this is amazing!

very very well done! very fun, and I'm debating whether I can already "play" this with my daughter :-D

**Pull request merged**